### ISSUE TITLE
Incorrect handling of exception during end transition

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#26246]
An IB tests showed that global end run work was still being done at the point where all modules are being deleted. In the job, an exception occurred during an end transition.

**Comment by Dr15Jones** [comment]
The log is
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc700/CMSSW_10_6_ROOT614_X_2019-03-24-2300/pyRelValMatrixLogs/run/23.0_JpsiMM+JpsiMMINPUT+DIGI+RECO+HARVEST/step3_JpsiMM+JpsiMMINPUT+DIGI+RECO+HARVEST.log#/5469-5469

with the relevant information being
```
%MSG-w HLTTauDQMOffline:  HLTTauDQMOfflineSource:hltTauOfflineMonitor_PFTaus  25-Mar-2019 05:15:00 CET Run: 1 Event: 2181
HLTTauDQML1Plotter::analyze: unable to read L1 met collection caloStage2Digis:EtSum
%MSG
25-Mar-2019 05:15:01 CET  Closed file file:step2.root
----- Begin Fatal Exception 25-Mar-2019 05:15:02 CET-----------------------
An exception of category 'LogicError' occurred while
   [0] Calling EventProcessor::runToCompletion (which does almost everything after beginJob and before endJob)
Exception Message:
In IndexIntoFile::sortVector_Run_Or_Lumi_Entries. A run entry is missing.
This means the IndexIntoFile product in the output file will be corrupted.
The output file will be unusable for most purposes.
If this occurs after an unrelated exception was thrown in
endLuminosityBlock or endRun then ignore this exception and fix
the primary exception. This is an expected side effect.
Otherwise please report this to the core framework developers
----- End Fatal Exception -------------------------------------------------

TrigReport ---------- Event  Summary ------------
TrigReport Events total = 100 passed = 100 failed = 0
```
and
```
Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning              1773                1773
Error                   3                   3
System                  5                   5

dropped waiting message count 977
pure virtual method called
terminate called without an active exception


A fatal system signal has occurred: abort signal
The following is the call stack containing the origin of the signal.

Mon Mar 25 05:15:02 CET 2019
Thread 6 (Thread 0x7f8087002700 (LWP 26877)):
#0  0x00007f81a2df7965 in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
#1  0x00007f81a33d7cec in __gthread_cond_wait (__mutex=<optimized out>, __cond=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_1_0_pre2-slc7_amd64_gcc700/build/CMSSW_10_1_0_pre2-build/BUILD/slc7_amd64_gcc700/external/gcc/7.0.0-omkpbe2/gcc-branches_gcc-7-branch-257125/obj/x86_64-unknown-linux-gnu/libstdc++-v3/include/x86_64-unknown-linux-gnu/bits/gthr-default.h:864
#2  std::condition_variable::wait (this=<optimized out>, __lock=...) at ../../../../../libstdc++-v3/src/c++11/condition_variable.cc:53
#3  0x00007f815f5f6fda in Eigen::ThreadPoolTempl<tensorflow::thread::EigenEnvironment>::WorkerLoop(int) () from /cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc700/cms/cmssw-patch/CMSSW_10_6_ROOT614_X_2019-03-24-2300/external/slc7_amd64_gcc700/lib/libtensorflow_framework.so
#4  0x00007f815f5f3aa7 in std::_Function_handler<void (), tensorflow::thread::EigenEnvironment::CreateThread(std::function<void ()>)::{lambda()#1}>::_M_invoke(std::_Any_data const&) () from /cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc700/cms/cmssw-patch/CMSSW_10_6_ROOT614_X_2019-03-24-2300/external/slc7_amd64_gcc700/lib/libtensorflow_framework.so
#5  0x00007f81a33dd8ff in std::execute_native_thread_routine (__p=0x7f813b10fc70) at ../../../../../libstdc++-v3/src/c++11/thread.cc:83
#6  0x00007f81a2df3dd5 in start_thread () from /lib64/libpthread.so.0
#7  0x00007f81a2b1cead in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x7f81705ff700 (LWP 26822)):
#0  0x00007f81a2b1220d in poll () from /lib64/libc.so.6
#1  0x00007f818e6a04d7 in full_read.constprop () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#2  0x00007f818e6a0b6c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#3  0x00007f818e6a1bd7 in sig_dostack_then_abort () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00007f81a2a55207 in raise () from /lib64/libc.so.6
#6  0x00007f81a2a568f8 in abort () from /lib64/libc.so.6
#7  0x00007f81a33b4cc5 in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00007f81a33b2a96 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:47
#9  0x00007f81a33b2ae1 in std::terminate () at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:57
#10 0x00007f81a33b383f in __cxxabiv1::__cxa_pure_virtual () at ../../../../libstdc++-v3/libsupc++/pure.cc:50
#11 0x00007f81a5e46ed5 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#12 0x00007f81a4135176 in tbb::internal::custom_scheduler<tbb::internal::IntelSchedulerTraits>::local_wait_for_all (this=0x7f81a02b3e00, parent=..., child=<optimized out>) at ../../src/tbb/custom_scheduler.h:521
#13 0x00007f81a412ea60 in tbb::internal::arena::process (this=0x7f81a03cad00, s=...) at ../../src/tbb/arena.cpp:160
#14 0x00007f81a412d3e3 in tbb::internal::market::process (this=0x7f81a03cb580, j=...) at ../../src/tbb/market.cpp:710
#15 0x00007f81a41295e3 in tbb::internal::rml::private_worker::run (this=0x7f81a01ee080) at ../../src/tbb/private_server.cpp:270
#16 0x00007f81a4129849 in tbb::internal::rml::private_worker::thread_routine (arg=<optimized out>) at ../../src/tbb/private_server.cpp:223
#17 0x00007f81a2df3dd5 in start_thread () from /lib64/libpthread.so.0
#18 0x00007f81a2b1cead in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x7f81717c0700 (LWP 26821)):
#0  0x00007f81a2ae3e2d in nanosleep () from /lib64/libc.so.6
#1  0x00007f81a2ae3cc4 in sleep () from /lib64/libc.so.6
#2  0x00007f818e69fda1 in sig_pause_for_stacktrace () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#3  <signal handler called>
#4  0x00007f81a2dfa4ed in __lll_lock_wait () from /lib64/libpthread.so.0
#5  0x00007f81a2df5dcb in _L_lock_883 () from /lib64/libpthread.so.0
#6  0x00007f81a2df5c98 in pthread_mutex_lock () from /lib64/libpthread.so.0
#7  0x00007f818cce3b81 in DQMStore::cloneRunHistograms(unsigned int, unsigned int) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libDQMServicesCore.so
#8  0x00007f8156420929 in non-virtual thunk to one::dqmimplementation::DQMRunEDProducer<>::endRunProduce(edm::Run&, edm::EventSetup const&) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginDQMOfflineTriggerPlugins.so
#9  0x00007f81a5f44155 in edm::one::EDProducerBase::doEndRun(edm::RunPrincipal const&, edm::EventSetupImpl const&, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#10 0x00007f81a5e70c40 in edm::WorkerT<edm::one::EDProducerBase>::implDoEnd(edm::RunPrincipal const&, edm::EventSetupImpl const&, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#11 0x00007f81a5e46785 in decltype ({parm#1}()) edm::convertException::wrap<bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}>(bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#12 0x00007f81a5e469a3 in bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#13 0x00007f81a5e46cd3 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(std::__exception_ptr::exception_ptr const*, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#14 0x00007f81a5e470c3 in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#15 0x00007f81a5e47181 in edm::SerialTaskQueue::QueuedTask<void edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#16 0x00007f81a4135176 in tbb::internal::custom_scheduler<tbb::internal::IntelSchedulerTraits>::local_wait_for_all (this=0x7f81a02bbe00, parent=..., child=<optimized out>) at ../../src/tbb/custom_scheduler.h:521
#17 0x00007f81a5ee8864 in edm::EventProcessor::endRun(edm::Hash<2> const&, unsigned int, bool, bool) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#18 0x00007f81a5ee8a8b in edm::EventProcessor::endUnfinishedRun(edm::Hash<2> const&, unsigned int, bool, bool, bool) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#19 0x00007f81a5ee8c0f in std::_Sp_counted_ptr_inplace<edm::(anonymous namespace)::RunResources, std::allocator<edm::(anonymous namespace)::RunResources>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#20 0x0000000000413786 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() ()
#21 0x00007f81a5ef0288 in std::_Sp_counted_ptr_inplace<edm::LuminosityBlockProcessingStatus, std::allocator<edm::LuminosityBlockProcessingStatus>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#22 0x0000000000413786 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() ()
#23 0x00007f81a5ee2999 in edm::FunctorWaitingTask<edm::EventProcessor::globalEndLumiAsync(edm::WaitingTaskHolder, std::shared_ptr<edm::LuminosityBlockProcessingStatus>)::{lambda(std::__exception_ptr::exception_ptr const*)#2}>::~FunctorWaitingTask() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#24 0x00007f81a4134cc7 in tbb::internal::custom_scheduler<tbb::internal::IntelSchedulerTraits>::local_wait_for_all (this=0x7f81a02bbe00, parent=..., child=<optimized out>) at ../../src/tbb/custom_scheduler.h:541
#25 0x00007f81a412ea60 in tbb::internal::arena::process (this=0x7f81a03cad00, s=...) at ../../src/tbb/arena.cpp:160
#26 0x00007f81a412d3e3 in tbb::internal::market::process (this=0x7f81a03cb580, j=...) at ../../src/tbb/market.cpp:710
#27 0x00007f81a41295e3 in tbb::internal::rml::private_worker::run (this=0x7f81a01ee180) at ../../src/tbb/private_server.cpp:270
#28 0x00007f81a4129849 in tbb::internal::rml::private_worker::thread_routine (arg=<optimized out>) at ../../src/tbb/private_server.cpp:223
#29 0x00007f81a2df3dd5 in start_thread () from /lib64/libpthread.so.0
#30 0x00007f81a2b1cead in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x7f81721c1700 (LWP 26820)):
#0  0x00007f81a2ae3e2d in nanosleep () from /lib64/libc.so.6
#1  0x00007f81a2ae3cc4 in sleep () from /lib64/libc.so.6
#2  0x00007f818e69fda1 in sig_pause_for_stacktrace () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#3  <signal handler called>
#4  0x00007f81a2dfa4ed in __lll_lock_wait () from /lib64/libpthread.so.0
#5  0x00007f81a2df5dcb in _L_lock_883 () from /lib64/libpthread.so.0
#6  0x00007f81a2df5c98 in pthread_mutex_lock () from /lib64/libpthread.so.0
#7  0x00007f818cce3b81 in DQMStore::cloneRunHistograms(unsigned int, unsigned int) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libDQMServicesCore.so
#8  0x00007f8085f9d3d9 in non-virtual thunk to one::dqmimplementation::DQMRunEDProducer<>::endRunProduce(edm::Run&, edm::EventSetup const&) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginValidationRecoMuon.so
#9  0x00007f81a5f44155 in edm::one::EDProducerBase::doEndRun(edm::RunPrincipal const&, edm::EventSetupImpl const&, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#10 0x00007f81a5e70c40 in edm::WorkerT<edm::one::EDProducerBase>::implDoEnd(edm::RunPrincipal const&, edm::EventSetupImpl const&, edm::ModuleCallingContext const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#11 0x00007f81a5e46785 in decltype ({parm#1}()) edm::convertException::wrap<bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}>(bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#12 0x00007f81a5e469a3 in bool edm::Worker::runModule<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#13 0x00007f81a5e46cd3 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >(std::__exception_ptr::exception_ptr const*, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::MyPrincipal const&, edm::EventSetupImpl const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3>::Context const*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#14 0x00007f81a5e470c3 in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#15 0x00007f81a5e47181 in edm::SerialTaskQueue::QueuedTask<void edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::RunPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#16 0x00007f81a4135176 in tbb::internal::custom_scheduler<tbb::internal::IntelSchedulerTraits>::local_wait_for_all (this=0x7f81a02cbe00, parent=..., child=<optimized out>) at ../../src/tbb/custom_scheduler.h:521
#17 0x00007f81a412ea60 in tbb::internal::arena::process (this=0x7f81a03cad00, s=...) at ../../src/tbb/arena.cpp:160
#18 0x00007f81a412d3e3 in tbb::internal::market::process (this=0x7f81a03cb580, j=...) at ../../src/tbb/market.cpp:710
#19 0x00007f81a41295e3 in tbb::internal::rml::private_worker::run (this=0x7f81a01ee100) at ../../src/tbb/private_server.cpp:270
#20 0x00007f81a4129849 in tbb::internal::rml::private_worker::thread_routine (arg=<optimized out>) at ../../src/tbb/private_server.cpp:223
#21 0x00007f81a2df3dd5 in start_thread () from /lib64/libpthread.so.0
#22 0x00007f81a2b1cead in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x7f818003e700 (LWP 26814)):
#0  0x00007f81a2dfb179 in waitpid () from /lib64/libpthread.so.0
#1  0x00007f818e69ffa7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#2  0x00007f818e6a0a8a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#3  0x00007f81a33dd8ff in std::execute_native_thread_routine (__p=0x7f8197ad1370) at ../../../../../libstdc++-v3/src/c++11/thread.cc:83
#4  0x00007f81a2df3dd5 in start_thread () from /lib64/libpthread.so.0
#5  0x00007f81a2b1cead in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x7f81a1155480 (LWP 26634)):
#0  0x00007f81a2ae3e2d in nanosleep () from /lib64/libc.so.6
#1  0x00007f81a2ae3cc4 in sleep () from /lib64/libc.so.6
#2  0x00007f818e69fda1 in sig_pause_for_stacktrace () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/pluginFWCoreServicesPlugins.so
#3  <signal handler called>
#4  0x00007f81a5ea7fec in std::_Sp_counted_ptr<edm::Worker*, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#5  0x00007f81a5ea80ea in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#6  0x00007f81a5ea8055 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#7  0x00007f81a5ea8055 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#8  0x00007f81a5ea8055 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::Worker> > > >*) () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#9  0x00007f81a5ea7484 in edm::WorkerRegistry::~WorkerRegistry() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#10 0x00007f81a5ee2662 in std::default_delete<edm::Schedule>::operator()(edm::Schedule*) const [clone .isra.942] [clone .part.943] () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#11 0x00007f81a5ee8dc7 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms-ib.cern.ch/nweek-02569/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_ROOT614_X_2019-03-24-0000/lib/slc7_amd64_gcc700/libFWCoreFramework.so
#12 0x000000000040e6e3 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#13 0x000000000040e17f in main ()

Current Modules:

Module: none (crashed)
Module: RecoMuonValidator:recoMuonVMuAssoc_tgt
Module: JetMonitor:PFJet80_Prommonitoring
Module: none

A fatal system signal has occurred: abort signal
```

**Comment by Dr15Jones** [comment]
assign core

**Comment by wddgit** [comment]
Recommend closing this. It should be fixed by PR #26289

**Comment by Dr15Jones** [comment]
+1
